tuning/tuner_requirements/requirements.txt

```diff
@@ -2,3 +2,4 @@ pytest==8.2.2
 tqdm==4.66.4
 z3_solver==4.13.0.0
 types-tqdm==4.66.0.20240417
+pre-commit==3.8.0
```

Comment:
We don't make the distinction in many of our other repos at the moment, but this could go in a `requirements-dev.txt`. That separates requirements needed to _use_ the project from requirements needed to _develop_ the project.